datasketch/lsh.py

```diff
@@ -226,6 +226,25 @@ def insert(
         """
         self._insert(key, minhash, check_duplication=check_duplication, buffer=False)
 
+    def merge(
+            self,
+            other: MinHashLSH,
+            check_overlap: bool = False      
+    ):
+        """Merge the other MinHashLSH with this one, making this one the union
+        of both the MinHashLSH.
+
```

Comment:
Need to mention what initialization parameters are checked, and what are not checked (e.g., `hash_func` is not checked).